src/orion/core/cli/frontend.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=too-few-public-methods
+"""
+Web application endpoint
+========================
+
+Starts an http endpoint to serve requests
+
+"""
+import logging
+
+from gunicorn.app.base import BaseApplication
+
+log = logging.getLogger(__name__)
+DESCRIPTION = "Starts Oríon Dashboard"
+
+
+import mimetypes
+import os
+
+import falcon
+
+
+class StaticResource:
+    """Resource class to serve frontend files."""
+
+    STATIC_DIR = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__), "..", "..", "..", "..", "dashboard", "build"
```

Comment:
Indeed, it may lead to issues. I added commits to:
- install dashboard build using setup.py `data_files` ption
- locate dashboard build at runtime, either in installation or in source code